core/ide/src/main/java/org/alice/stageide/sceneeditor/StorytellingSceneEditor.java

```diff
@@ -555,17 +574,19 @@ private boolean isSelectableType(AbstractType<?, ?, ?> valueType) {
   private void initializeCameraMarkers() {
     PerspectiveCameraMarker openingSceneMarker = new PerspectiveCameraMarker();
     openingSceneMarker.setColorId(Color.DARK_GRAY);
-    this.openingSceneMarkerImp = EmployeesOnly.getImplementation(openingSceneMarker);
-    this.openingSceneMarkerImp.setDisplayVisuals(true);
-    MarkerUtilities.addIconForCameraImp(this.openingSceneMarkerImp, "mainCamera");
-    MarkerUtilities.setViewForCameraImp(this.openingSceneMarkerImp, View.STARTING_CAMERA_VIEW);
-
-    PerspectiveCameraMarker sceneViewMarker = new PerspectiveCameraMarker();
-    sceneViewMarker.setColorId(Color.LIGHT_BLUE);
-    this.sceneViewMarkerImp = EmployeesOnly.getImplementation(sceneViewMarker);
-    this.sceneViewMarkerImp.setDisplayVisuals(true);
-    MarkerUtilities.addIconForCameraImp(this.sceneViewMarkerImp, "sceneEditorCamera");
-    MarkerUtilities.setViewForCameraImp(this.sceneViewMarkerImp, View.LAYOUT_SCENE_VIEW);
+    this.startingCameraMarkerImp = EmployeesOnly.getImplementation(openingSceneMarker);
+    this.startingCameraMarkerImp.setDisplayVisuals(true);
+    this.startingCameraMarkerImp.setCameraType(STARTING_CAMERA);
+    MarkerUtilities.addIconForCameraImp(this.startingCameraMarkerImp, "mainCamera");
+    MarkerUtilities.setViewForCameraImp(this.startingCameraMarkerImp, View.STARTING_CAMERA_VIEW);
```

Comment:
If you set a View on each MarkerImp the second argument here is not needed. Not much of a simplification, but an option.

Reply:
Why does the second argument here is not needed?

Reply:
If you change `setCameraType()` to take a `View` (or whatever it gets called) enum instead of a number, and do that for all the MarkerImps, the markerImp passed in to `setViewForCameraImp` can be queried with `getCameraType()` instead of passing it in.
This is perhaps moe an example of of some possible benefit in using the enum instead of an int.

Reply:
Got it! we should use some benefits of the enum.